src/common/services/api/profile.service.js

```diff
@@ -29,7 +29,7 @@ class Profile{
     return this.cortexApiService.get({
         path: ['profiles', this.cortexApiService.scope, 'default'],
         zoom: {
-          paymentMethods: 'paymentmethods:element[]'
+          paymentMethods: 'paymentmethods:element[],paymentmethods:element:bankaccount,paymentmethods:element:bankaccount:element,paymentmethods:element:creditcard,paymentmethods:element:creditcard:element'
```

Comment:
These aren't used for checkout but I imagine they'll be used in self service for managing gifts and other things. What do you think of making 2 service methods? `getPaymentMethods` and something like `getPaymentMethodsWithDonations`. That query is kinda slow...